### DISCUSSION TITLE
Menu Config-Events->Tab Config -Sampling "when get it fired"

### DISCUSSION THREAD

**Discussion by ju-emb** [artisan-roaster-scope/artisan#1687] [General]
Hello all,
Artisan 3.0.3

Menu Config-Events->Tab Config  under Sampling 
I select WebSocket Command and "read( {{ "command": "readOutput", "params": {{ "Output": 1 }}}} ) "

select sync or 1s

observing what's going into my WebSocketServer, I don't see that command get fired

What I'm doing wrong

thanks
Ju

**Comment by ju-emb** [comment]
Sorry tipo
Artisan 3.0.0

**Comment by MAKOMO** [comment]
Try with single curly brackets like

`read({ "command": "readOutput", "params": {"Output": 1 }})`

**Reply by ju-emb** [reply]
That worked,

many thanks Marko

**Comment by ju-emb** [comment]
Solved the Problem With Marko's help

thanks Marko
